httpevents/request.go

```diff
@@ -42,37 +42,38 @@ func makeRequest(req *http.Request, laddr string) request {
 }
 
 func (r *request) log(logger *events.Logger, depth int) {
+	var mem [10]interface{}
+	var arg = append(mem[:0], r.laddr, r.raddr, r.host, r.method)
```

Comment:
it's using the `mem` array as backend for the `arg` slice. It could also be written as `append(make([]interface{}, 0, 10), ...)` but it's more verbose.